blacklight-cornell/app/controllers/application_controller.rb

```diff
@@ -23,55 +23,49 @@ class ApplicationController < ActionController::Base
 
 # An array of strings to be added to HTML HEAD section of view.
 # See ApplicationHelper#render_head_content for details.
-   def extra_head_content
-       #Deprecation.warn Blacklight::LegacyControllerMethods, "#extra_head_content is NOT deprecated"
-        @extra_head_content ||= []
-   end
+  def extra_head_content
+    #Deprecation.warn Blacklight::LegacyControllerMethods, "#extra_head_content is NOT deprecated"
+    @extra_head_content ||= []
+  end
 
-   def show_login_action
-     Rails.logger.info("es287_debug #{__FILE__}:#{__LINE__} logging in before hook")
-   end
 
+  def show_login_action
+    # :nocov:
+      Rails.logger.info("es287_debug #{__FILE__}:#{__LINE__} logging in before hook")
+    # :nocov:
+  end
 
 protected
   def authenticate_user!
-    Rails.logger.info("es287_debug #{__FILE__}:#{__LINE__} authenticate user")
     if user_signed_in?
-      Rails.logger.info("es287_debug #{__FILE__}:#{__LINE__} authenticate user call super")
       super
     else
-      Rails.logger.info("es287_debug #{__FILE__}:#{__LINE__} authenticate redirect to saml ")
       if ENV['SAML_IDP_TARGET_URL']
         #redirect_to 'http://es287-dev.library.cornell.edu:8988/saml.html'
         redirect_to request.base_url + '/saml.html'
       end
-      #redirect_to 'http://es287-dev.library.cornell.edu:8986/users/auth/saml'
-      #redirect_to new_user_session_path, :notice => 'if you want to add a notice'
-      ## if you want render 404 page
-      #      ## render :file => File.join(Rails.root, 'public/404'), :formats => [:html], :status => 404, :layout => false
     end
   end
 
+  #TODO: I'm puzzled as to why we have this here? From what I understand this
+  # will never be defined since it's in scope of "if false". Should we remove this?
+  # ****************************************************************************
   if false
```

Comment:
Does anyone have insight into why we have this here? It doesn't seem like it is ever defined due to the `if false` statement

Reply:
This was committed 8 years ago with the commit message "SAML - do not lose context": https://github.com/cul-it/blacklight-cornell/commit/0500c1d6a2114fa87118a72235a2b8b341828513. It's in version control – you can get rid of it.